backend/app/utils/s3.py

```
@@ -0,0 +1,109 @@
+
+        except Exception as e:
+            print(f"An unexpected error occurred: {str(e)}")
+
+        return None  # Return None if download failed
+
+    def upload_to_s3(self,file_path, bucket_name, object_name):
+        """Upload a file to s3"""
+        try:
+            self.s3_class_client.upload_file(file_path, bucket_name, object_name)
+            print(f"Uploaded {file_path} to {bucket_name}/{object_name}")
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)